pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -629,43 +636,47 @@ private static void setupApiActivityFilter() {
         });
     }
 
-    private static void setupMDC(String path, Route route) {
-        pathToClass.put(path, route.getClass().getSimpleName());
-        before(path, (request, response) -> MDC.put(MDC_ROUTE_CLASS, pathToClass.get(path)));
+    private static void setupMDC(String path, Route route, String operation) {
```

Comment:
Nitpick: can we reorder args so path and operation (Let's call `operation` **`httpMethod`** and `path` **`urlPath`**

Reply:
Fixed. But I better leave name `path` because it is used in all other methods - just to easier understand that it is about same `path`.

Reply:
Did you reorder the method arguments to path, operation, route?

Reply:
Reordered